packages/react-components/react-teaching-popover-preview/etc/react-teaching-popover-preview.api.md

```diff
@@ -27,29 +47,35 @@ export const renderTeachingPopoverCarousel_unstable: (state: TeachingPopoverCaro
 export const renderTeachingPopoverHeader_unstable: (state: TeachingPopoverHeaderState) => JSX.Element;
 
 // @public
-export const renderTeachingPopoverPageCount_unstable: (state: TeachingPopoverPageCountState) => JSX.Element;
+export const renderTeachingPopoverPageCount_unstable: (state: TeachingPopoverPageCountState) => any;
```

Comment:
`any` worries there, curious why it happens.

(let's fill an item to check)

Reply:
This is due to the popover page count allowing a render function (that provides current/total page number), or a direct ReactNode as children, will see if I can define this as a type

Reply:
Fixed, missed a root on one of the return styles.